src/components/ContributorsSection.test.tsx

```diff
@@ -5,35 +5,74 @@ import { ContributorsSection } from "./ContributorsSection";
 
 describe("ContributorsSection", () => {
   it("renders a 'Contributors' heading", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "github:dbartholomae/test-repo",
-    };
-
     expect(
-      await render(<ContributorsSection title="Contributors" pkg={pkg} />)
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          title="Contributors"
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
     ).toContain("## Contributors");
 
     expect(
-      await render(<ContributorsSection pkg={pkg} />)
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
     ).toContain("## ‍💼 Contributors");
   });
 
   it("renders sentence with a link to the contributors on GitHub", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "github:dbartholomae/test-repo",
-    };
-    expect(await render(<ContributorsSection pkg={pkg} />)).toContain(
+    expect(
+      await render(
+        <ContributorsSection
+          contributors={[]}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
+    ).toContain(
       "This package only works thanks to [all of our contributors](https://github.com/dbartholomae/test-repo/graphs/contributors)"
     );
   });
 
-  it("renders nothing when repository is not hosted on GitHub", async () => {
-    const pkg = {
-      name: "package-name",
-      repository: "bitbucket:user/test-repo",
-    };
-    expect(await render(<ContributorsSection pkg={pkg} title="Contributors" />)).toBe("")
+  it("renders a list of contributors on GitHub", async () => {
+    const contributors = [
+      {
+        id: 1,
+        name: "dbartholomae",
+      },
+    ];
+    expect(
+      await render(
+        <ContributorsSection
+          contributors={contributors}
+          owner="dbartholomae"
+          repo="test-repo"
+        />
+      )
+    ).toContain(
+      '[![dbartholomae](https://avatars.githubusercontent.com/u/1?size=100 "dbartholomae")](https://github.com/dbartholomae "dbartholomae\'s GitHub profile\'")'
```

Comment:
The sizing of the avatars doesn't seem to work correctly as you can see when you scroll to the README of the branch:
https://github.com/junwen-k/jsx-readme/tree/feat/%2347

Reply:
Do you have any recommendation to workaround this issue? I've did some research on other contributor list and they usually involve some HTML code. As far as I know we are using `jsx-md` and it only outputs markdown without any raw HTML.

Ideally, my thinking is that the component should dynamically spans how many avatars in one row based on the number of contributors, but using only Markdown it seems like a challenge. I've tried using `<Table />` component from `jsx-md` and it seems like doesn't work very well in this case.

The way I try to hack it now is by explicitly setting the size of the avatar using GitHub avatar server, but apparently it does not work if there's no custom avatar image provided. See comments on https://stackoverflow.com/a/36380674.

See [all-contributors](https://github.com/all-contributors/all-contributors) and [markdown-contributors](https://thodorisbais.github.io/markdown-contributors/) for example.

Reply:
Hi, it's fine to use html and create a `HtmlImg` component in this library. I might move it to the other library at some point, but for now let's not limit the functionality of `jsx-readme` by only using components from `jsx-md`. In the end, components can just return text.

Reply:
I see, I will try to implement that :) Not sure if I know how to do it or not though. Thank you!

Reply:
Something along these lines:
```tsx
// in src/elements/HtmlImage.tsx

/* @jsx MD */
import MD, { Component, Fragment } from "..";

/** @internal */
interface Props {
  children: string;
  src: string;
}

/**
 * @example
 *   ```js
 *   render(<HtmlImage src="./image.png" title="Title">
 *     Alternative text
 *   </Image>)
 *   ===
 *   '<img alt="Alternative text" src="./image.png">'
 */
export const HtmlImage: Component<Props> = ({ children, src }) => (
  <Fragment>
    {'<img alt="'}
    {children}
    {'" src="'}
    {src}
    {'">'}
  </Fragment>
);
```
(based on the [jsx-md image component](https://github.com/dbartholomae/jsx-md/blob/main/src/elements/Image.tsx), you can also add a [test](https://github.com/dbartholomae/jsx-md/blob/main/src/elements/Image.test.tsx) to confirm that it works)

Reply:
Or, if you need more complicated things with tables etc. (similar to all-contributors), you can add a `Contributor` component and add whatever HTML you need.